library/alloc/src/vec/into_iter.rs

```diff
@@ -315,6 +315,13 @@ impl<T, A: Allocator> FusedIterator for IntoIter<T, A> {}
 #[unstable(feature = "trusted_len", issue = "37572")]
 unsafe impl<T, A: Allocator> TrustedLen for IntoIter<T, A> {}
 
+#[stable(feature = "default_iters", since = "1.64.0")]
+impl<T> Default for IntoIter<T> {
```

Comment:
suggestion: put a doc comment on all these impls (or the method, I forget which we do) to *say* what they do.

Reply:
Done